android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java

```diff
@@ -210,39 +214,53 @@ private void applyMessageStyle(@NonNull Context context, NotificationCompat.Buil
 
-            // Conversational styling should be applied to groups chats, rooms, and any 1:1 chats with more than one notification (ensuring the large profile image is always shown)
-            if (!conversationName.isEmpty() || hasExistingNotification) {
-                // Create the messaging style notification builder for this notification, associating it with the person who sent the report comment
-                NotificationCompat.MessagingStyle messagingStyle = new NotificationCompat.MessagingStyle(person)
-                        .setGroupConversation(true)
-                        .setConversationTitle(conversationName);
+            NotificationCompat.MessagingStyle messagingStyle = new NotificationCompat.MessagingStyle(person);
 
+            // Add all conversation messages to the notification, including the last one we just received.
+            List<NotificationCompat.MessagingStyle.Message> messages;
+            if (hasExistingNotification) {
+                NotificationCompat.MessagingStyle previousStyle = NotificationCompat.MessagingStyle.extractMessagingStyleFromNotification(existingReportNotification.getNotification());
+                messages = previousStyle != null ? previousStyle.getMessages() : new ArrayList<>(List.of(recreatePreviousMessage(existingReportNotification)));
+            } else {
+                messages = new ArrayList<>();
+            }
 
-                // Add all conversation messages to the notification, including the last one we just received.
-                List<NotificationCompat.MessagingStyle.Message> messages;
-                if (hasExistingNotification) {
-                    NotificationCompat.MessagingStyle previousStyle = NotificationCompat.MessagingStyle.extractMessagingStyleFromNotification(existingReportNotification.getNotification());
-                    messages = previousStyle != null ? previousStyle.getMessages() : new ArrayList<>(List.of(recreatePreviousMessage(existingReportNotification)));
-                } else {
-                    messages = new ArrayList<>();
-                }
-
-                // add the last one message we just received.
-                messages.add(newMessage);
+            // add the last one message we just received.
+            messages.add(newMessage);
 
-                for (NotificationCompat.MessagingStyle.Message activeMessage : messages) {
-                    messagingStyle.addMessage(activeMessage);
-                }
+            for (NotificationCompat.MessagingStyle.Message activeMessage : messages) {
+                messagingStyle.addMessage(activeMessage);
+            }
 
-                builder.setStyle(messagingStyle);
+            // Conversational styling should be applied to groups chats, rooms, and any 1:1 chats with more than one notification (ensuring the large profile image is always shown)
+            if (!conversationName.isEmpty()) {
```

Comment:
```suggestion
            if (!roomName.isEmpty()) {
```

Can you rename this variable to make it a little more clear?

Reply:
Changed 🫡